flocker/ca/_ca.py

```diff
@@ -95,8 +99,8 @@ def create_certificate_authority(keypair, dn, request, serial, expiry, digest):
 
     :param int serial: The certificate serial number.
 
-    :param int expiry: Number of seconds from now until this certificate
-        should expire.
+    :param str expiry: ASN1 formatted datetime string representing the
```

Comment:
Bytes. `str` should only be used to describe a Python-related object, e.g. something that is used with `getattr()`. Otherwise always use `bytes` or `unicode`, since `str` is ambiguous (it means Unicode on Python 3).